pygef/gef.py

```diff
@@ -742,6 +766,12 @@ def __init__(self, header_s, data_s):
             )
         )
 
+        if old_column_names:
```

Comment:
Should ` elevation_with_respect_to_NAP` also be in the mapping?

Reply:
In that constructor it's not added..